src/components/artwork_filter/index.tsx

```diff
@@ -31,9 +32,22 @@ class ArtworkFilter extends React.Component<Props, State> {
       dimension_range: "*",
       price_range: "*",
       medium: "*",
+      loading: false,
     }
   }
 
+  handleLoadMore() {
+    this.setState({ loading: true }, () => {
+      this.props.relay.setVariables({
+        size: this.props.relay.variables.size + PageSize,
+      }, readyState => {
+        if (readyState.done) {
+          this.setState({ loading: false })
+        }
+      })
+    })
```

Comment:
This could maybe be simplified by checking for `pendingVariables` where ever the `loading` variable would be used (not seeing it being used btw, what purpose does it serve?): https://facebook.github.io/relay/docs/api-reference-relay-container.html#pendingvariables

I think that by switching to that you can get rid of both `setState` calls.